sdks/java/io/rrio/src/test/java/org/apache/beam/io/requestresponseio/CallShouldBackoffBasedOnRejectionProbabilityTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.io.requestresponseio;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for {@link CallShouldBackoffBasedOnRejectionProbability}. */
+@RunWith(JUnit4.class)
+public class CallShouldBackoffBasedOnRejectionProbabilityTest {
+
+  @Test
+  public void testValue() {
```

Comment:
Please add additional tests for:

* No requests or exceptions.
* Many requests no exceptions.
* Many requests all exceptions.

Reply:
`No requests or exceptions.` is already tested with test case `Case.of(0, false)` in `CallShouldBackoffBasedOnRejectionProbabilityTest`. For many requests no exceptions v all exceptions, I am going to add more trues and falses respectively to the existing cases.